assets/src/dashboard/app/views/templates/detail.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useEffect, useState, useContext } from 'react';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import useRouteHistory from '../../router/useRouteHistory';
+import { ApiContext } from '../../api/apiProvider';
+import { TransformProvider } from '../../../../edit-story/components/transform';
+import FontProvider from '../../font/fontProvider';
+import { CardGallery, PreviewPage } from '../../../components';
+
+const PageContainer = styled.div`
+  display: flex;
+  padding: 20px;
+
+  @media ${({ theme }) => theme.breakpoint.largeDisplayPhone} {
+    display: block;
+  }
+`;
+
+const PageColumn = styled.div`
+  display: flex;
+  justify-content: center;
+  width: 50%;
+
+  @media ${({ theme }) => theme.breakpoint.largeDisplayPhone} {
+    width: 100%;
+  }
+`;
+
+function TemplateDetail() {
+  const [template, setTemplate] = useState(null);
+  const {
+    state: {
+      queryParams: { id: templateId },
+    },
+  } = useRouteHistory();
+  const {
+    actions: { fetchTemplate },
+  } = useContext(ApiContext);
+
+  useEffect(() => {
+    async function getTemplateById(id) {
+      setTemplate(await fetchTemplate(parseInt(id)));
+    }
+
+    getTemplateById(templateId);
+  }, [fetchTemplate, templateId]);
+
+  return (
+    <PageContainer>
+      <PageColumn>
+        {template && (
+          <FontProvider>
+            <TransformProvider>
+              <CardGallery>
+                {template.pages.map((page) => (
+                  <PreviewPage key={page.id} page={page} />
+                ))}
+              </CardGallery>
+            </TransformProvider>
+          </FontProvider>
+        )}
+      </PageColumn>
+      <PageColumn />
```

Comment:
Coming soon, this will hold the Template details (i.e. Title, description, colors, etc...).